photo_mapper_webserver/authentication/api.py

```
@@ -0,0 +1,70 @@
+    try:
+        access_token, refresh_token = refresh_access_token(refresh_token)
+
+        response = HttpResponse(
+            content='{"message": "Refreshed"}',
+            content_type="application/json",
+            status=200,
+        )
+
+        set_auth_cookies(response, access_token, refresh_token)
+    
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)